docs/installation/prepare/microcontrollers/development-setup.md

```diff
@@ -222,75 +222,18 @@ Navigate to [the Viam app](https://app.viam.com) and [add a new robot](/manage/f
 Click on the name of the robot to go to the robot's page.
 Then, navigate to the **Config** tab.
 
-### Configure an esp32 board
+### Configure your robot with an esp32
```

Comment:
```suggestion
### Configure your robot with an ESP32
```
[nit] As above I do think this looks odd since the user won't know we're talking about the model name string, so better to just use the normal name of the thing

Reply:
Sounds good, done